### PR TITLE
Crime Assist pda program (DeltaV)

### PR DESCRIPTION
**Pull request by dvir001**
## About the PR
Add a Crime Assist program, which helps NFSD and Lawyer players determine what crime was committed without having to have a wiki open.

## Why / Balance
For many players, evidently, reading up on Space Law is an insurmountable obstacle, and as a result they have trouble identifying what is and isn't a crime. For this reason, a PDA program that gives a basic way to see what sort of crime was committed may help.

## Technical details
The PDA program is fairly simple. It has a ew buttons (Start, Home, Yes, No) which are visible/invisible depending on the program's State. This State also determines which Localization string is displayed.

The laws are based on DeltaV and will need changes done for Frontier to be fully working.

## Media
https://github.com/DeltaV-Station/Delta-v/pull/464

## Breaking changes
N/A

**Changelog**
:cl: dvir01
- add: Added the Crime Assist program to Security and Lawyer PDA's, to help identify a crime with a simple yes/no question tree.

### CODE REVIEW

**Comment by dvir001** [comment]
This is fully working, but using DeltaV laws right now
Ill need someone to help fix the yml to Frontier space law.

File: Resources/Prototypes/_Nyano/Entities/Objects/Devices/pda.yml [hunk 1]
```diff
@@ -44,15 +44,25 @@
 
 - type: entity
   parent: BasePDA
-  id: GuardPDA
-  name: guard PDA
+  id: PrisonGuardPDA
```
Comment (Master2112) [inline Resources/Prototypes/_Nyano/Entities/Objects/Devices/pda.yml:47]: You'll need to revert this change; the PrisonGuardPDA is a DeltaV thing

File: Resources/Prototypes/_Nyano/Entities/Objects/Devices/pda.yml [hunk 2]
```diff
@@ -44,15 +44,25 @@
...
+  name: prison guard PDA
   description: Red to hide the stains of prisoner blood.
   components:
   - type: Pda
-    id: GuardIDCard
+    id: PrisonGuardIDCard
```
Comment (Master2112) [inline Resources/Prototypes/_Nyano/Entities/Objects/Devices/pda.yml:52]: Same for this line; the linter is failing on this because **Prison**GuardIDCard doesn't exist

File: Resources/Prototypes/_Nyano/Entities/Objects/Devices/pda.yml (hunk 1 above)
Comment (dvir001) [reply Resources/Prototypes/_Nyano/Entities/Objects/Devices/pda.yml:47]: Dont worry im adding it to fix it.
